source/upgrade/important-upgrade-notes.rst

```diff
@@ -18,6 +18,94 @@ Important Upgrade Notes
 | If you’re upgrading                                | Then...                                                                                                                                                          |
 | from a version earlier than...                     |                                                                                                                                                                  |
 +====================================================+==================================================================================================================================================================+
+| v10.7                                              | Added new column ``BannerInfo`` in ``Channels`` table for storing metadata for an upcoming licensed feature. The migration is fully backwards-compatible,        |
+|                                                    | non-locking, and zero downtime is possible. Below are the SQL queries included in the schema changes:                                                            |
+|                                                    |                                                                                                                                                                  |
+|                                                    | **PostgreSQL**:                                                                                                                                                  |
+|                                                    |                                                                                                                                                                  |
+|                                                    | .. code-block:: sql                                                                                                                                              |
+|                                                    |                                                                                                                                                                  |
+|                                                    | ALTER TABLE channels ADD COLUMN IF NOT EXISTS bannerinfo jsonb;                                                                                                  |
```

Comment:
Any lines of code following a ``..code-block:: sql`` directive needs to be indented in order to display as expected in generated output.